portal-web/test/functional/com/liferay/portalweb/paths/portlet/assetpublisher/APConfiguration.path

```diff
@@ -260,6 +260,11 @@
 	<td>//div[contains(.,'Asset Entries')]/following-sibling::div//a[@title='Select']</td>
 	<td></td>
 </tr>
+<tr>
+	<td>ASSET_SELECTION_MANUAL_ASSET_ENTRIES_SELECT_IN_DEPOT</td>
+	<td>//div[contains(.,'Asset Entries')]/following-sibling::div//a[@title='Select in Test Depot Name']</td>
```

Comment:
@liuxingye this should be more flexible. I've moved the hardcoding to the macro level, but if we're going to be writing more tests using different AL names then we should change this at the macro level as well.

Reply:
yeah, noticed your changes, thanks @austinchiang